packages/shared/src/contexts/NotificationsContext.tsx

```diff
@@ -32,6 +33,10 @@ export const NotificationsContextProvider = ({
     globalThis.window?.Notification?.permission === 'granted',
   );
 
+  const notificationsAvailable = () => {
+    return !!globalThis.window?.Notification;
```

Comment:
I added a simple check here to see if the `Notification` object is present in the current environment, we can improve this if we need.